ansible_base/authentication/utils/authentication.py

```diff
@@ -21,6 +21,10 @@ def setting(self, *args, **kwargs):
         return ["username", "email"]
 
 
+def raise_auth_exception(message):
+    raise AuthException(None, message)
```

Comment:
Any reason for why we aren't making any best-effort attempts at fetching the authentication backend?

Reply:
I'm not sure having it makes a ton of difference but I pushed a change to try and get that.